content/staff-wellbeing.html.erb

```diff
@@ -0,0 +1,149 @@
+
+            <%= render('/partials/resource_card.*', title: "Secondary school staff wellbeing statement", href: "#",
+              body: "Read a wellbeing statement developed to effectively support school staff.", tag: "Example", details: {
+              reading_time: "3 minutes", created_by: "St. Peter's School (Secondary)" }) %>
+          </ul>
+        </div>
+
+        <div class="govuk-grid-row dfe-width-container govuk-!-padding-top-6 govuk-!-padding-bottom-0">
+          <h2 class="govuk-heading-s">Decide who will drive it forward</h2>
+        </div>
+
+        <div class="govuk-grid-row dfe-width-container">
+          <ul class="resource-card-group">
+
+            <%= render('/partials/resource_card.*', title: "Establishing a wellbeing committee", href: "#",
+              body: "Learn how to set up a wellbeing committee.", tag: "Example", details: {
+              reading_time: "4 minutes", created_by: "Notre Dame High School" }) %>
+
+            <%= render('/partials/resource_card.*', title: "Establishing a school workload group", href: "#",
+              body: "Learn how to set up a workload group.", tag: "Example", details: {
+              reading_time: "20 minutes", created_by: "Baylis Court School (Secondary)" }) %>
+
+            <%= render('/partials/resource_card.*', title: "Prioritising wellbeing to improve staff retention", href: "#",
+              body: "Learn how prioritising wellbeing impacted staff retention.", tag: "Example", details: {
+              reading_time: "11 minutes", created_by: "Bedford Academy (Secondary)" }) %>
+
+            <%= render('/partials/resource_card.*', title: "Prioritising wellbeing to improve staff retention", href: "#",
+              body: "Learn how prioritising wellbeing impacted staff retention.", tag: "Example", details: {
+              reading_time: "11 minutes", created_by: "Bedford Academy (Secondary)" }) %>
+
+          </ul>
+        </div>
+
+        <div class="govuk-grid-row dfe-width-container govuk-!-padding-top-6 govuk-!-padding-bottom-0">
+          <h2 class="govuk-heading-s">Take action</h2>
+        </div>
+
+        <div class="govuk-grid-row dfe-width-container">
+          <ul class="resource-card-group">
+
+            <%= render('/partials/resource_card.*', title: "21 things we did to improve staff wellbeing", href: "#",
+              body: "Read advice and tips on improving staff wellbeing.", tag: "Example", details: {
+              reading_time: "3 minutes", created_by: "King Charles I Secondary School" }) %>
+
+            <%= render('/partials/resource_card.*', title: "Whole school approach to wellbeing", href: "#",
+              body: "Read advice and tips on improving staff wellbeing.", tag: "Example", details: {
+              reading_time: "7 minutes", created_by: "Barr Beacon School (Secondary)" }) %>
+
+          </ul>
+        </div>
+      </div>
+      <div class="govuk-grid-column-one-third govuk-!-padding-top-0 govuk-!-padding-left-6">
```

Comment:
I created the `two-column-page` class so that we didn't need to add left/right padding to the columns - do we still need `govuk-!-padding-left-6` here?